src/main/java/com/dm/dmservicesandbox/controller/UserVerificationController.java

```diff
@@ -0,0 +1,36 @@
+package com.dm.dmservicesandbox.controller;
+
+import com.dm.dmservicesandbox.component.UserVerificationComponent;
+import com.dm.dmservicesandbox.domain.ServiceReponse;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@Slf4j
+@RestController
+public class UserVerificationController {
+
+    @Autowired
+    UserVerificationComponent userVerificationComponent;
+
+    //registration confirmation API
+    private final static String REGISTER_CONF_APIPATH = "/dm/registerConfirmation";
+
+    //redo email verification API
+    private final static String REDO_VERIFY_APIPATH = "/dm/redoVerification";

```

Comment:
shall we create a single class for these  API Paths constants. you have one in token class also

Reply:
Yeah agree. That's what I wanted to do but don't know how to do :) 
How about I merge those changes as is and you do some refactoring. I think you can do it better since you are more familiar with this framework :)